pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -257,21 +257,21 @@ func (m *ManagerImpl) PluginDisconnected(resourceName string) {
 // is captured. Also, registered device and device to container allocation
 // information is checkpointed to the disk.
 func (m *ManagerImpl) PluginListAndWatchReceiver(resourceName string, resp *pluginapi.ListAndWatchResponse) {
-	var devices []pluginapi.Device
+	var devices []*pluginapi.Device
 	for _, d := range resp.Devices {
-		devices = append(devices, *d)
+		devices = append(devices, d)
 	}
 	m.genericDeviceUpdateCallback(resourceName, devices)
```

Comment:
can this simplify to `m.genericDeviceUpdateCallback(resourceName, resp.Devices)`?

Reply:
Makes sense, changed as suggested.